### ISSUE TITLE
[FE] - Add visual cue to Profile Settings

### ISSUE THREAD

**Issue by marcelluscaio** [bluewave-labs/Checkmate#1121]
In the profile settings, the tabs (Profile, Password, Teams) do not provide visual cues for the focus state, making harder for keyboard navigation. Also, the border is misaligned.

There should be a visual cue when navigating between these interactive elements. It would be best if they follow these guidelines:
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html
https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html

I don't see an alternative implementation.

**Comment by guninwasan** [comment]
Hey @marcelluscaio, can I work on this issue?

**Comment by marcelluscaio** [comment]
Thank you for your interest, @guninwasan ! Yes, you can work on that

**Comment by guninwasan** [comment]
@marcelluscaio just wanted to get some more clarity on the issue. I tried keyboard navigation on profile both locally and the demo site, and it seems like there are visual cues to show what the tab the user is on
<img width="236" alt="Screenshot 2024-11-19 at 21 19 49" src="https://github.com/user-attachments/assets/78ea1846-072a-432f-ac59-b21a83ed73b8">
<img width="236" alt="Screenshot 2024-11-19 at 21 20 07" src="https://github.com/user-attachments/assets/dab3ed35-c6a9-4788-9067-f9f3755c7c71">

**Comment by gorkem-bwl** [comment]
@guninwasan just wanted to check back the progress here. Thanks!

**Comment by guninwasan** [comment]
Hey @gorkem-bwl and @marcelluscaio the [PR](https://github.com/bluewave-labs/checkmate/pull/1232) is up for review. Let me know if I need to make any changes to it. Thanks!
